README.md

```diff
@@ -1,72 +1,81 @@
-# telegram ecommerce
+# 🤖 Telegram e-commerce
 
-This is a telegram bot made using the python language. To connect with the telegram API I use the [python-telegram-bot](https://python-telegram-bot.org/) wrapper and on the database side, I use MySQL.
+This is a Telegram bot made using the Python language. To connect with the Telegram API, I use the python-telegram-bot wrapper, and on the database side, I use MySQL.
+## 💖 All Contributors
 
+<a href="https://github.com/raulpy271/telegram_ecommerce/graphs/contributors">
```

Comment:
It's nice to have a contributors section, but seems that you git accout isn't setup properly, because your avatar isn't showing up in this section. 

I think you can solve it by adding your name and the email of your github account in your local git config. See this [tutorial](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).

Reply:
ok :) will try to fix that